pg_proctab.c

```diff
@@ -16,20 +16,54 @@
 #include <sys/stat.h>
```

Comment:
Admittedly pgnodemx.c is getting a bit too long, but in any case the other "proc" functions are all defined in there. I think for this first addition of "proctab" functions we should keep everything in pgnodemx.c for the sake of consistency. A later cleanup might be good which moves all of the "proc" functions to their own source file (perhaps procfuncs.c?).

Reply:
Fair enough. Once it works I can move it back into pgnodemx.c